### PR TITLE
B side of Alpha 4 changes

### PR DESCRIPTION
**Pull request by jkilpatr**
This removes things slated to be removed in this release.

### CODE REVIEW

File: rita/src/rita_common/network_endpoints/mod.rs [hunk 1]
```diff
@@ -53,25 +53,20 @@ pub fn hello_response(
     their_id: Json<serde_json::Value>,
```
Comment (kingoflolz) [inline rita/src/rita_common/network_endpoints/mod.rs]: we can change this to Json<Identity>

File: rita/src/rita_common/network_endpoints/mod.rs [hunk 2]
```diff
@@ -53,25 +53,20 @@ pub fn hello_response(
...
-    let new_id: Result<Identity, serde_json::Error> = serde_json::from_value(their_id.clone());
-    let old_id: Result<LocalIdentity, serde_json::Error> = serde_json::from_value(their_id.clone());
+    let id: Result<Identity, serde_json::Error> = serde_json::from_value(their_id.clone());
 
-    match new_id {
+    match id {
         Ok(new_id) => {
             info!("sending to new hello handler");
-            hello_response_new(new_id, req)
+            hello_response_handler(new_id, req)
```
Comment (kingoflolz) [inline rita/src/rita_common/network_endpoints/mod.rs]: maybe we should inline this?

**Comment by kingoflolz** [comment]
I think there is one thing I forgot to comment, try searching for `get_future()`, we need to make the future behavior the normal behavior

File: rita/src/rita_common/network_endpoints/mod.rs (hunk 2 above)
Comment (jkilpatr) [reply rita/src/rita_common/network_endpoints/mod.rs]: should we just remove hello_response in general and do the deserialization in the calling function?

**Comment by jkilpatr** [comment]
not seeing anything else with get future.

**Comment by jkilpatr** [comment]
backwards compat changes tested manually works well.

**Pull request merged**